pkgs/top-level/python-packages.nix

```diff
@@ -8751,6 +8751,8 @@ in {
     };
   };
 
+  pepper = callPackage ../development/python-modules/pepper { };
```

Comment:
@FRidh I can put it on pkgs/tools/admin/pepper/default.nix or pkgs/tools/admin/salt/pepper.nix. The second one looks best to me (but I would actually have create a "saltstack" repo instead of "salt). What do you think ?